src/Apps/__stories__/OrderApp.story.tsx

```diff
@@ -24,7 +25,15 @@ const Router = props => (
 )
 
 storiesOf("Apps/Order Page/Offer", module).add("Empty", () => (
-  <Router initialRoute="/orders/123/offer" />
+  <Router
+    initialRoute="/orders/123/offer"
+    mockResolvers={mockResolver({
+      ...UntouchedOfferOrder,
+      mode: "OFFER",
+      requestedFulfillment: null,
+      state: "PENDING",
```

Comment:
#superminor I think you don't need to overwrite `mode` and `state` here, since they have the same values in `UntouchedOfferOrder`.